### PR TITLE
Update issue templates - Closes #1188

### LINKED ISSUE #1188: Create issue templates & update contribution guidelines

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1188

### CODE REVIEW

**Review by shuse2 — COMMENTED** [review]
I think all repositories are following the same format
docs repo can also align on the same format?
https://github.com/LiskHQ/lisk-sdk/tree/development/.github/ISSUE_TEMPLATE

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
about: Create a report to help us further improve.
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: ''
+labels: 'type: bug'
+assignees: ''
+
+---
+**Expected content**
+A clear and concise description of what content you expect to see.
+
+**Actual content**
+Describe the current documentation content that is buggy.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
Describe the specific content of the documentation that you feel could be improved.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for thee Lisk documentation
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
about: Suggest a recommendation for improving the Lisk documentation.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+title: ''
+labels: 'type: feature'
+assignees: ''
+
+---
+
+**Description**
+Please describe what functionality is needed.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
Please describe what functionality is required.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+**Motivation**
+Please describe why it is needed.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
Please describe why it is required.
```

File: .github/ISSUE_TEMPLATE/update-request.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+name: Update request
+about: Describe this issue template's purpose here.
```
Comment (chris529) [inline .github/ISSUE_TEMPLATE/update-request.md]:
```suggestion
about: Describe the objective and reason for this issue here.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

File: .github/ISSUE_TEMPLATE/update-request.md (hunk 6 above)
Comment (Tschakki) [reply .github/ISSUE_TEMPLATE/update-request.md]:
```suggestion
about: Report outdated content in the documentation.
```

**Review by chris529 — DISMISSED** [review]
LGTM!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us further improve.
+title: ''
+labels: 'type: bug'
+assignees: ''
+
+---
+**Expected content**
```
Comment (shuse2) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
### Expected content
```
We use h3 `###` in general

**Review by shuse2 — CHANGES_REQUESTED** [review]
just minor point, LGTM

**Pull request merged**